### ISSUE TITLE
Rendering a mesh with multiple textures saved in jpg files

### ISSUE THREAD

**Issue by AndreAmaduzzi** [ptrvilya/blendify#5]
Hello,
I am rendering ShapeNet models. Some of them have multiple textures saved in .jpg files.
How to include all of them in a blendify Mesh?

Thanks,
Andrea

**Comment by vguzov** [comment]
Hello Andrea,
That's a cool idea! Unfortunately, Blendify doesn't support multiple textures for single mesh as of now, but we can add this in the future versions. Could you please tell us, why exactly those meshes have multiple textures? Do they consist of several materials or each of those textures denote some material property (e.g. metallic, shiny, etc.)? If you can provide the example of such mesh and textures, that would be even better!

**Comment by AndreAmaduzzi** [comment]
Hello Vladimir,
I'm glad to read your quick response. I am referring to ShapeNet models. Some of them have multiple materials, each of them associated with a specific jpg file specifying its texture.
There are many examples of such models in the dataset. Some of them may be:
class: 03001627, model_id: 3ff879e891701079ea07bd8096c66582
class: 03001627, model_id: 3ff4401d898b488a601104cd2d998272

**Comment by ptrvilya** [comment]
Hey Andrea,

We have just released an updated version of Blendify. Now the framework can handle multiple materials and colors defined per face for a single mesh. We have added an [example](https://github.com/ptrvilya/blendify/blob/main/examples/08_per_face_material.py) that demonstrates it on the donut mesh. With this feature one should be able to render ShapeNet meshes correctly. Better late than never, I guess :D
